source/content.js

```diff
@@ -1,317 +1,100 @@
-		const ajaxContainer = select('#js-repo-pjax-container,#js-pjax-container');
-		if (ajaxContainer) {
-			ajaxContainer.append(<has-rgh/>);
-		}
-	});
-}
-
-// eslint-disable-next-line complexity
-function ajaxedPagesHandler() {
-	enableFeature(hideEmptyMeta);
-	enableFeature(removeUploadFilesButton);
-	enableFeature(addTitleToEmojis);
-	enableFeature(shortenLinks);
-	enableFeature(linkifyCode);
-	enableFeature(addDownloadFolderButton);
-	enableFeature(linkifyBranchRefs);
-	enableFeature(batchOpenIssues);
-	enableFeature(hideUselessComments);
-	enableFeature(navigatePagesWithArrowKeys);
-	enableFeature(makeHeadersSticky);
-
-	if (pageDetect.isGlobalIssueSearch() || pageDetect.isGlobalPRSearch()) {
-		enableFeature(addYoursMenuItem);
-		enableFeature(addCommentedMenuItem);
-	}
-
-	enableFeature(sortIssuesByUpdateTime); // Must be after addYoursMenuItem + addCommentedMenuItem
-
-	if (pageDetect.isMilestone()) {
-		enableFeature(addMilestoneNavigation); // Needs to be before sortMilestonesByClosestDueDate
-	}
-
-	if (pageDetect.isRepo()) {
-		enableFeature(hideReadmeHeader);
-		enableFeature(addBranchButtons);
-		enableFeature(addDiffViewWithoutWhitespaceOption);
-		enableFeature(addCILink);
-		enableFeature(sortMilestonesByClosestDueDate); // Needs to be after addMilestoneNavigation
-		enableFeature(addStarRepoHotkey);
-	}
-
-	if (pageDetect.isRepoRoot()) {
-		enableFeature(addToggleFilesButton);
-	}
-
-	if (pageDetect.isPR()) {
-		enableFeature(scrollToTopOnCollapse);
-		enableFeature(addDeleteForkLink);
-		enableFeature(fixSquashAndMergeTitle);
-		enableFeature(fixSquashAndMergeMessage);
-		enableFeature(openCIDetailsInNewTab);
-		enableFeature(waitForBuild);
-		enableFeature(hideInactiveDeployments);
-		enableFeature(addPullRequestHotkey);
-		enableFeature(addQuickReviewButtons);
-	}
-
-	if (pageDetect.isPR() || pageDetect.isIssue()) {
-		enableFeature(linkifyIssuesInTitles);
-		enableFeature(embedGistInline);
-		enableFeature(extendStatusLabels);
-		enableFeature(highlightClosingPrsInOpenIssues);
-		enableFeature(addOPLabels);
-		enableFeature(addTimeMachineLinksToComments);
-	}
-
-	if (pageDetect.isIssue() || pageDetect.isPRConversation()) {
-		enableFeature(addJumpToBottomLink);
-	}
-
-	if (pageDetect.isIssueList()) {
-		enableFeature(addFilterCommentsByYou);
-		enableFeature(hideIssueListAutocomplete);
-		enableFeature(excludeFilterShortcut);
-	}
-
-	if (pageDetect.isIssueList() || pageDetect.isPR() || pageDetect.isIssue()) {
-		enableFeature(showRecentlyPushedBranchesOnMorePages);
-	}
-
-	if (pageDetect.isReleasesOrTags()) {
-		enableFeature(addCreateReleaseShortcut);
-	}
-
-	if (pageDetect.isCommit()) {
-		enableFeature(addPatchDiffLinks);
-	}
-
-	if (pageDetect.isCompare()) {
-		enableFeature(addSwapBranchesOnCompare);
+// Must be called after all the features were added to safeOnAjaxedPages
+// to mark the current load as "done", so history.back() won't reapply the same DOM changes
+features.safeOnAjaxedPages(() => {
```

Comment:
This needs to be renamed to `onAjaxedPages`
```suggestion
features.onAjaxedPages(() => {
```